src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRunner.cs

```diff
@@ -48,6 +49,8 @@ public static async Task<IReadOnlyList<RestoreSummary>> RunAsync(RestoreArgs res
             RestoreArgs restoreArgs,
             CancellationToken token)
         {
+            X509TrustStore.Initialize(restoreArgs.Log);
```

Comment:
>X509TrustStore.Initialize(restoreArgs.Log); [](http://example.com/codeflow?start=12&length=43)

What is the perf impact of this?

Reply:
See https://github.com/NuGet/NuGet.Client/pull/4585#discussion_r871876349